### PR TITLE
DT-6360 add navigator view content

### PR DESCRIPTION
**Pull request by vesameskanen**
New component which tracks journey progress. At this point, some instructions for moving to stop/park/rental are given.

Figma layouts are not final and component rendering is not pixel ccurate.

### CODE REVIEW

File: app/component/itinerary/Navigator.js [hunk 1]
```diff
@@ -1,10 +1,34 @@
...
-import { intlShape } from 'react-intl';
+import { FormattedMessage, intlShape } from 'react-intl';
 import { createFragmentContainer, graphql } from 'react-relay';
 import { itineraryShape } from '../../util/shapes';
-import { legTime } from '../../util/legUtils';
+import { legTime, legTimeStr } from '../../util/legUtils';
 import Icon from '../Icon';
+import NaviLeg from './NaviLeg';
+
+/*
```
Comment (Antiik91) [inline app/component/itinerary/Navigator.js:10]: Is this necessary?

File: app/component/itinerary/Navigator.js [hunk 2]
```diff
@@ -1,10 +1,34 @@
...
+        end {
+          scheduledTime
+          estimated {
+            time
+          }
+        }
+      }
+    }
+  }
+`;
```
Comment (Antiik91) [inline app/component/itinerary/Navigator.js:30]: Is this necessary?

File: app/component/itinerary/NaviLeg.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+          <div>
+            {stop?.name}&nbsp;
+            {stop?.code && <StopCode code={stop.code} />}
+            {rentalVehicle?.rentalNetwork.networkId}
+            {vehicleParking?.name}
+            {vehicleRentalStation?.rentalNetwork.networkId}&nbsp;
+            {vehicleRentalStation?.name}
+          </div>
+          <div onClick={() => focusToLeg(leg, false)} className="navileg-focus">
+            Näytä reitti kartalla
```
Comment (Antiik91) [inline app/component/itinerary/NaviLeg.js]: this should be translated

File: app/component/itinerary/Navigator.js (hunk 2 above)
Comment (vesameskanen) [reply app/component/itinerary/Navigator.js:30]: will be :)

**Pull request merged**